build/template-build-and-run-cachecompat-tests.yaml

```diff
@@ -18,11 +18,26 @@ steps:
 - task: UseDotNet@2
   displayName: 'Use .Net Core sdk 5.x'
   inputs:
-    version: 5.x
+    version: 6.x
+    includePreviewVersions: true
 
 # Install keyvault secrets
 - template: template-install-keyvault-secrets.yaml
 
+- task: DotNetCoreCLI@2
+  displayName: 'dotnet workload restore'
```

Comment:
maybe adding the project name will help uniquely identify the task, the below one has the same name